docs/05-components/modal.mdx

```diff
@@ -4,12 +4,58 @@ description: Grafana Labs Modal component
 draft: true
```

Comment:
@Clarity-89 @staton-hyse11 Do I have to delete the `draft: true` line in order to get this documentation published in Saga? Otherwise, I think it will not be shown 🤔

Reply:
You can either delete that line, or set `draft: false`, I think both should work.